Ihor.Skrypnyk/5/html/index.html

```diff
@@ -0,0 +1,154 @@
+          excepturi facilis fuga in ipsa nam nostrum odit perspiciatis sapiente sequi soluta sunt. Lorem ipsum dolor sit
+          amet, consectetur adipisicing elit. Ex id maxime officia qui temporibus. Accusantium, adipisci animi autem eveniet
+          itaque libero molestiae nulla repellat sint velit! Doloremque incidunt molestias quaerat. Lorem ipsum dolor sit
+          amet, consectetur adipisicing elit. Aliquid aut, autem corporis cupiditate, dignissimos distinctio esse eum excepturi
+          facilis fuga in ipsa nam nostrum odit perspiciatis sapiente sequi soluta sunt.</p>
+      </div>
+        <!-- should be floated right -->
+        <img class="rightFloat-star" src="../images/tile.png" />
+        <p> Here is some monospaced code block </p>
+        <!--Make it nice with padding and a little grey background and borders.-->
+        <pre class="left-formula">document.addEventListener('click', () => console.log('topkek'))
+        </pre>
+      </article>
+    </main>
+
+    <!--two sections. On left this links on right social buttons (steal some images from internet)-->
+    <footer>
+      <section class="footer--left">
+        <button class="footer-loginButton" onclick="document.getElementById('modal-wrapper').style.display='block'"> Login </button>
+        <a href="aboutUs.html"> About us </a>
+        <a href="contactUs.html"> Contact us </a>
+      </section>
+      <section class="footer--right">
+        <!-- should be a aligned to the right social network buttons -->
+        <a href="#">
+          <img class="footer--right logos" src="https://shop.az/site/img/svg/facebook-round.svg" />
+        </a>
+        <a href="#">
+          <img class="footer--right logos" src="http://badges.instagram.com/static/images/ig-badge-32.png" />
+        </a>
+        <a href="#">
+          <img class="footer--right logos" src="http://pngimg.com/uploads/twitter/twitter_PNG34.png" />
+        </a>
+      </section>
+    </footer>
+    <!-- Modal Login form -->
+
+  <div id="modal-wrapper" class="modal">
+
+    <form class="modal-content animate">
+
+      <div class="imgcontainer">
+        <span class="close" title="Close PopUp">&times;</span>
+        <h1 style="text-align:center">Login to the website</h1>
+      </div>
+
+      <div class="modal-container">
+        <input type="text" placeholder="Enter Username" name="uname">
+        <input type="password" placeholder="Enter Password" name="psw">
+        <button class="modal-button" type="submit">Login</button>
+        <input type="checkbox" style="margin:26px 30px;"> Remember me
+        <a href="#" style="text-decoration:none; float:right; margin-right:34px; margin-top:26px;">Forgot Password ?</a>
+      </div>
+
+    </form>
+  </div>
+  <script>
+    let modal = document.getElementById('modal-wrapper');
+    let close = document.querySelector('.close');
```

Comment:
This is dangerous, cause you can get more than 2 elements with this class name, so it's better to use and id or data-id attribute for more reliable selection.